### PR TITLE
Clean up a few overlooked old-school tests

### PR DESCRIPTION
**Pull request by abayer**
cc @reviewbybees esp @rsandell

### CODE REVIEW

File: pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BasicModelDefTest.java [hunk 1]
```diff
@@ -159,9 +159,7 @@ public void parallelPipeline() throws Exception {
 
     @Test
     public void executionModelAction() throws Exception {
-        prepRepoWithJenkinsfile("executionModelAction");
-
-        WorkflowRun b = getAndStartBuild();
+        WorkflowRun b = expect("executionModelAction").go();
         j.assertBuildStatusSuccess(j.waitForCompletion(b));
```
Comment (rsandell) [inline pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BasicModelDefTest.java]: `go()` should have already run the build, waited for completion and asserted success.

Comment (abayer) [inline pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BasicModelDefTest.java]: whoops!

**Comment by rsandell** [comment]
:bee:

**Comment by rsandell** [comment]
:bee:

**Pull request merged**